workflow.src

```diff
@@ -0,0 +1,121 @@
+{
+  "title":  "Workflow for editors and other contributors"
+}
+---
+<main>
+      <p>
+        A good practice for GitHub is to use <a href="https://help.github.com/articles/about-pull-requests/">pull requests</a>.
+        It lets you tell others about changes you've pushed to the repository.
+        Once a pull request is opened, you can discuss and review the potential changes with
+        others and add follow-up commits before the changes are merged into the repository.
+      </p>
+  <ol>
+    <li>
+      <p>
+        Navigate to the repo.
+      </p>
+      <pre><code>cd specs-on-github</code></pre>
+    </li>
+    <li>
+      <p>
+        Checkout the main branch of your repository (in general, this is the <code>master</code> branch).
+      </p>
+      <pre><code>git checkout master</code></pre>
+    </li>
+    <li>
+      <p>
+        Make sure the main branch is up-to-date with upstream.
+      </p>
+      <pre><code>git pull</code></pre>
+    </li>
+    <li>
+      <p>
+        Create a new branch for your upcoming pull request (one branch per pull request). Please make branch names
+        informative - by including the issue or bug number for example.
+      </p>
+      <pre><code>git checkout -B my-wonderful-edits-for-87</code></pre>
+    </li>
+    <li>
+      <p>
+        Make your edits in your favorite editor.
+      </p>
+    </li>
+    <li>
+      <p>
+        Add your edits to prepare your commit by staging them. Best is to use the interactive mode (New files still
+        need to be added explictly).
+      </p>
+      <pre><code>git add -p</code></pre>
+      <p>If you ignore an edit, it will be reproposed to you the next time you run the command. This allow you to make
```

Comment:
I'm not sure what "reproposed" means here?

Reply:
Also, I've never seen `-p`... maybe explain what that does? 

Nit: s/allow/allows

Reply:
`-p` is for partial add - when you only want to commit a subset of the changes you brought to a given file

Reply:
Since someone showed me the -p 2 years ago, I never use anything else to prepare my commits. I found it extremely useful. If it feels too advanced, I guess I can make a version of the doc for simple usage, and a separate one for more advanced users.